app/views/expenditures_with_circ_stats_reports/new.html.erb

```diff
@@ -17,16 +17,16 @@
   <%= render 'shared/funds_selection', f: f %>
   <%= render 'shared/date_range_selection', f: f %>
```

Comment:
The drop-down menu to select dates is not aligned with the rest of the form but the fund selection is aligned. I'm assuming it would look better aligned.
![Screen Shot 2020-03-04 at 10 51 00 AM](https://user-images.githubusercontent.com/6343464/75912551-42197200-5e06-11ea-8090-adba53e58aec.png)